packages/mock-account-provider/app/routes/accounts.$accountId.tsx

```diff
@@ -0,0 +1,48 @@
+import { json } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+import { getAccountTransactions } from '../lib/transactions.server'
+import tableStyle from '../styles/table.css'
+
+type LoaderData = {
+  transactions: Awaited<ReturnType<typeof getAccountTransactions>>
+}
+
+export const loader = async ({ params }) => {
+  return json<LoaderData>({
+    transactions: await getAccountTransactions(params.accountId)
+  })
+}
+
+export default function Transactions() {
+  const { transactions } = useLoaderData() as LoaderData
+  return (
+    <main>
+      <h1>Transactions</h1>
+      <table>
+        <tr>
+          <th>Date</th>
+          <th>Type</th>
+          <th>Amount</th>
+          <th>Description</th>
+        </tr>
+        {transactions.map((trx) => (
+          <tr key={trx.id}>
+            <td>{trx.createdAt}</td>
+            <td>{trx.sendAmountValue ? 'Outgoing' : 'Incoming'}</td>
+            <td>
+              {(
+                Number(trx.sendAmountValue ?? trx.incomingAmountValue) / 100
```

Comment:
Do we want to show `receivedAmountValue` at all here?

Just for my own understanding: `incomingAmountValue` is the amount originally the incoming payment was created for, but  `receivedAmountValue` was the actual amount transferred?

Reply:
Also, as an aside, can I partially fulfill an incoming payment, create a "partial" quote, create an outgoing payment, then try creating a quote again for the same incoming payment?

Reply:
> Do we want to show `receivedAmountValue` at all here?

Yes, this should use it instead of `incomingAmountValue` because :point_down: 
> Just for my own understanding: `incomingAmountValue` is the amount originally the incoming payment was created for, but `receivedAmountValue` was the actual amount transferred?

Yes, and `incomingAmount` is optional, an incoming payment can be created without it.

>Also, as an aside, can I partially fulfill an incoming payment, create a "partial" quote, create an outgoing payment, then try creating a quote again for the same incoming payment?

Yes

Reply:
I agree, we should use `sentAmount` and `receivedAmount`.